test/blackbox-tests/test-cases/dune-cache/cache-disabled.t

```diff
@@ -0,0 +1,130 @@
+Set up project and cache directory
+
+  $ echo "(lang dune 3.20)" > dune-project
+ 
+  $ cat > config <<EOF
+  > (lang dune 3.20)
+  > (cache enabled)
+  > EOF
+
+  $ cat > dune << EOF
+  > (library
+  >  (name foo))
+  > EOF
+
+  $ cat > foo.ml << EOF
+  > let f x y = x + y
+  > EOF
+
+  $ export DUNE_CACHE_ROOT=$(pwd)/dune_test_cache
+  $ mkdir $DUNE_CACHE_ROOT
+
+Initial build, populating cache
+
+  $ dune build --config-file config
+  $ wc -l _build/log 
```

Comment:
I would have preferred to `cat _build/log` but some lines are starting with `$` and it confuses the cram tests on subsequent runs.